plio/permissions.py

```diff
@@ -0,0 +1,34 @@
+from rest_framework import permissions
+from organizations.middleware import OrganizationTenantMiddleware
+from plio.settings import DEFAULT_TENANT_SHORTCODE
+from users.models import OrganizationUser
+
+
+class PlioPermission(permissions.BasePermission):
+    """
+    Permission check for plios.
+    """
+
+    def has_permission(self, request, view):
+        return True
+
+    def has_object_permission(self, request, view, obj):
+        if request.user.is_superuser:
+            return True
+
+        organization_shortcode = (
+            OrganizationTenantMiddleware.get_organization_shortcode(request)
+        )
+        if organization_shortcode == DEFAULT_TENANT_SHORTCODE:
```

Comment:
how does this work for the `play` endpoint? In that case, we should be granting read access, no?

Reply:
also, once it's done here, we won't need to do it in the `get_queryset` function right?

Reply:
For the /play/ endpoint, the `has_object_permission` is not called. That method is not called for list views. This is only called when a put/post/patch request is called.
In the /play/ endpoint, we're not doing any putting/posting/patching. we're only listing. So for that case it skips this `has_object_permission`
But if someone is editing a plio, in that case this method is called

Reply:
And no, we'll have to do it in get_queryset function as well.
Because in get_queryset, we need to filter plios in a different way depending on the organization shortcode and other conditions